perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -2483,8 +2483,13 @@ public void deleteAttribute(PerunSession sess, AttributeDefinition attribute) th
 		//Remove services' required attributes
 		//TODO
 
-		AttributeDefinition attributeDef = new AttributeDefinition(attribute);
+		//Remove attribute and all it's values
+		getPerunBl().getAuditer().log(sess,new AttributeDeleted(attribute));
```

Comment:
We generally log auditer message after the call to impl layer, please move it after ``getAttributesManagerImpl().deleteAttribute(sess, attribute);``

Reply:
Fixed.